### ISSUE TITLE
Add option to change zoombox hotkey

### ISSUE THREAD

**Issue by jamesmugford** [rishikapadia7/AvraLabsEyeControl#7]
It would be nice to have an option to change the hotkey for triggering zoombox.

This would allow for compatibility with other clicking tools such as the fantastic project: Gaze OCR

**Comment by rishikapadia7** [comment]
Yes that could be worthwhile. We could add additional hooks inside PrecisiongGazeMouseForm and then tie that into a settings option. The UI for configuring the hotkey will require quite some work. See if anyone is up for it.
